packages/gatsby-plugin-vercel-analytics/README.md

```diff
@@ -21,7 +21,7 @@ npm i @vercel/gatsby-plugin-vercel-analytics
 or
 
 ```bash
-yarn add @vercel/gatsby-plugin-vercel-analytics
+pnpm add @vercel/gatsby-plugin-vercel-analytics
```

Comment:
Lets either add all options (npm/yarn/pnpm) to the readme, or just have npm only